projects/ngrx.io/content/guide/router-store/configuration.md

```diff
@@ -68,35 +68,16 @@ export const reducers: ActionReducerMap&lt;State&gt; = {
 };
 </code-example>
 
-**In your AppModule**
-
-<code-example header="app.module.ts">
-@NgModule({
-  imports: [
-    StoreModule.forRoot(reducers),
-    RouterModule.forRoot([
-      // routes
-    ]),
-    StoreRouterConnectingModule.forRoot({
-      serializer: CustomSerializer
-    })
-  ]
-})
-export class AppModule {}
-</code-example>
-
-### Using the Standalone API
-
-<code-example header="main.ts">
-import { bootstrapApplication } from '@angular/platform-browser';
+<code-example header="app.config.ts">
```

Comment:
Above code example we should add a subtitle as follows:

```md
**In your application config**

<code-example header="app.config.ts">
...
```

Reply:
Done